envisage/plugins/ipython_kernel/internal_ipkernel.py

```diff
@@ -61,6 +68,13 @@ def init_ipkernel(self, gui_backend):
         # Start IPython kernel with GUI event loop support
         self.ipkernel = gui_kernel(gui_backend)
 
+        # Since ipykernel 4.7, the io_loop attribute of the kernel is not
+        # initialized anymore
```

Comment:
Please could we have a reference to the issue here (in the format enthought/envisage#107)?

Reply:
done